app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -56,63 +40,32 @@ import org.mozilla.fenix.compose.Image
 import org.mozilla.fenix.home.recenttabs.RecentTab
 import org.mozilla.fenix.theme.FirefoxTheme
 
-/**
- * A list of recent tabs to jump back to.
- *
- * @param recentTabs List of [RecentTab] to display.
- * @param onRecentTabClick Invoked when the user clicks on a recent tab.
- * @param onRecentSearchGroupClicked Invoked when the user clicks on a recent search group.
- */
-@Composable
-fun RecentTabs(
```

Comment:
I am curious why this was moved. I would imagine we want to keep the public function at the top of the file since it's the main function we care about looking at `RecentTabs.kt`. I would otherwise try to avoid too much refactoring to reduce the amount of code that needs to be reviewed in a single commit.

Reply:
Not sure how that happened either 🤔 reverted that change